packages/now-build-utils/src/fs/node-version.ts

```diff
@@ -5,8 +5,13 @@ import debug from '../debug';
 const supportedOptions: NodeVersion[] = [
   { major: 12, range: '12.x', runtime: 'nodejs12.x' },
   { major: 10, range: '10.x', runtime: 'nodejs10.x' },
-  { major: 8, range: '8.10.x', runtime: 'nodejs8.10' },
-];
+  {
+    major: 8,
+    range: '8.10.x',
+    runtime: 'nodejs8.10',
+    discontinueDate: new Date('2020-01-01'),
+  },
+].filter(o => !isDiscontinued(o));
```

Comment:
Why was it added when it's filtered out immediately?

Reply:
Nevermind, I just realised that it will only be removed after `discontinueDate`

Reply:
The filter will only remove Node 8 in January. So yes it will be filtered out immediately starting then so that we don't need to push another update over the holiday.